src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -2203,8 +2203,10 @@ private BoundExpression BindIncrementOperator(CSharpSyntaxNode node, ExpressionS
         /// <summary>
         /// Returns false if reported an error, true otherwise.
         /// </summary>
-        private bool CheckConstraintAndRuntimeSupportForOperator(SyntaxNode node, MethodSymbol? methodOpt, TypeSymbol? constrainedToTypeOpt, BindingDiagnosticBag diagnostics)
+        private bool CheckConstraintLanguageVersionAndRuntimeSupportForOperator(SyntaxNode node, MethodSymbol? methodOpt, TypeSymbol? constrainedToTypeOpt, BindingDiagnosticBag diagnostics)
         {
+            bool result = true;
+
             if (methodOpt?.ContainingType?.IsInterface == true)
```

Comment:
nit: consider combining this check with the one below to reduce the overall level of nesting.